### ISSUE TITLE
The ground / platform should be more visible.

### ISSUE THREAD

**Issue by ngokevin** [supermedium/moonrider#69]
The platform acts as a frame of reference to reduce motion sickness, but while playing, the ground is not visible. Perhaps it can be shifted forward more to be at least visible in periphery.

**Comment by ngokevin** [comment]
Should we move the two feet triangle towards the back in the `home.obj` so we can nudge the platform forward a bit?

**Comment by DannyFGitHub** [comment]
I agree with moving the platform forward, as the on the Oculus Go it feels like the timing on which it should the block should be hit is too close to you.
However it should also be considered that the saber needs to be able get to the corners of the playing field for a Oculus Go controller.
